coreneuron/mechanism/mech/mod2c_core_thread.hpp

```diff
@@ -89,15 +92,17 @@ struct SparseObj {          /* all the state information */
     int do_flag;
 };
 
-#pragma acc routine seq
+nrn_pragma_acc(routine seq)
```

Comment:
`nrn_pragma_omp(declare target)` not needed here?

Reply:
I tried to follow the approach of only adding it when its absence caused problems. I guess this isn't actually called in the test suite 🤷

Reply:
I added it anyway.